Update readme about keys (#8)
hashgraph/hedera-network-e2e: README.md

1. Ensure to use Docker Compose version 1.29.2 on macOS, due to known bug in Docker Compose V2. 
2. Ensure the `gRPC FUSE for file sharing` and `Use Docker Compose V2` settings are disabled in the docker settings.

![docker-compose-settings.png](docker-compose-settings.png)

> &#10008; The keys under `network-node`(`hedera.key`, `hedera.crt` and `keys folder`) are only intended to be used for testing on local network. These keys should not be used against any other networks for testing. 

Reviewer: ```suggestion
&#10008; The keys under `network-node` (`hedera.key`, `hedera.crt` and the `keys` folder) are only intended to be used for testing with this docker based local network. These keys should not be used with any other networks. 
```
